documentation/staking-governance.md

```diff
@@ -29,21 +29,37 @@ The voting power for a locked amount of tokens is computed as follows:
 - Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.
 - In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%.
```

Comment:
Optional: Perhaps explain further the 200% e.g.,

```suggestion
- In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%, by pausing a maximum locking period of 2 years.
```

Reply:
I took your suggestion with small modifications. I think the 200% should be clear now. 
But I additionally modified the section a bit to introduce the token symbol `vpLSK` which is also used below.